app/routes/contests.ts

```diff
@@ -15,6 +16,10 @@ export default class ContestsRoute extends BaseRoute {
   @service declare router: RouterService;
   @service declare store: Store;
 
+  buildRouteInfoMetadata(): RouteInfoMetadata {
+    return new RouteInfoMetadata({ isDarkRoute: true });
```

Comment:
Nice! Didn't know this was a thing.

Reply:
Me neither, it was a draft RFC for a long time, then finally got accepted & merged at some point